pyblish_bumpybox/plugins/collect_existing_files.py

```diff
@@ -27,9 +30,52 @@ def version_get(self, string, prefix):
         matches = re.findall(regex, string, re.IGNORECASE)
         if not len(matches):
             msg = "No \"_"+prefix+"#\" found in \""+string+"\""
-            raise ValueError(msg)
+            self.log.debug(msg)
+            return None
         return matches[-1:][0][1], re.search("\d+", matches[-1:][0]).group()
 
+    def scan_versions(self, instance_collection, version):
```

Comment:
This should be a static method, given the call to scan_collection can also be static.